docs/04-operation-guides/troubleshooting/obsv-01-troubleshoot-prometheus-istio-server-crash-oom.md

```diff
@@ -29,42 +29,42 @@ For both solutions, you can choose to change your Kyma cluster settings or direc
 
 1. To increase the memory limit, create a values YAML file with the following content:
 
-```yaml
-monitoring:
-  prometheus-istio:
-    server:
-      resources:
-        limits:
-          memory: "6Gi"
-```
-
-> **TIP:** You should be fine with increasing the limit to 6Gi. But if your resources are scarce, try increasing the value gradually in steps of 1Gi.
+  ```yaml
+  monitoring:
+    prometheus-istio:
+      server:
+        resources:
+          limits:
+            memory: "6Gi"
+  ```
+  
+  > **TIP:** You should be fine with increasing the limit to 6Gi. However, if your resources are scarce, try increasing the value gradually in steps of 1Gi.
```

Comment:
This indentation should do the trick. 
``````suggestion
   ```yaml
   monitoring:
     prometheus-istio:
       server:
         resources:
           limits:
             memory: "6Gi"
   ```
  
   > **TIP:** You should be fine with increasing the limit to 6Gi. However, if your resources are scarce, try increasing the value gradually in steps of 1Gi.
``````

Reply:
I indented 2 blanks but should have been 4?